html/elements/isindex.json

```diff
@@ -1,148 +0,0 @@
-{
-  "html": {
-    "elements": {
-      "isindex": {
```

Comment:
Evidence for `<isindex>` removal is in https://github.com/mdn/browser-compat-data/pull/6931#discussion_r517262407.